sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -208,6 +209,7 @@ public int hashCode() {
     public static final Set<TypeName> STRING_TYPES = ImmutableSet.of(STRING);
     public static final Set<TypeName> DATE_TYPES = ImmutableSet.of(DATETIME);
     public static final Set<TypeName> CONTAINER_TYPES = ImmutableSet.of(ARRAY);
+    public static final Set<TypeName> MAP_TYPES = ImmutableSet.of(MAP);
```

Comment:
<!--new_thread; commit:7bf9d2048e8d568ee9280629db391775cc490358; resolved:0-->
should we consider this part of CONTAINER_TYPES?

Reply:
would separate here, CONTAINER should be ARRAY/SET. `List<KV<>>` could be a CONTAINER_TYPE, `Map<>` is not.